pkgs/leak_tracker_flutter_testing/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.0.0
```

Comment:
Can we just retract 1.0.3 and re-release with this, or has it already been picked up externally?

Reply:
Let's try.

It is in use in Flutter Framework. Yes, I will need to handle breaking changes anyway.
I am little afraid to get some side effects of version resolution.